server/src/main/kotlin/suwayomi/tachidesk/manga/MangaAPI.kt

```diff
@@ -69,6 +69,7 @@ object MangaAPI {
 
             get("{mangaId}/chapters", MangaController.chapterList)
             post("{mangaId}/chapter/batch", MangaController.chapterBatch)
```

Comment:
please move this to `/api/v1/chapter(s)/batch` if you don't want to put it into v2

Reply:
I'm not sure I understood what you ment. Please check the change.